catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -150,10 +150,9 @@ private void extractThumbnail(Metacard metacard, InputStream input) throws IOExc
 
         ClassLoader originalContextClassLoader = Thread.currentThread()
                 .getContextClassLoader();
-        try {
+        try (SlideShow<?, ?> genericSlideShow = SlideShowFactory.create(input)){
             Thread.currentThread()
                     .setContextClassLoader(getClass().getClassLoader());
```

Comment:
You might need to move this classloader part outside of the try. Depends on if that SlideShowFactory.create() call is the thing that is trying to do classloader stuff or not.

Reply:
I couldn't exactly tell... which means I probably shouldn't have adjusted the order. Will update 👍